src/frontend/Komponenter/Migrering/TabellInfotrygdPerioder.tsx

```diff
@@ -64,25 +64,25 @@ const formatStønadTom = (periode: InfotrygdPeriodeMedFlereEndringer): string =>
     }
 };
 
-const utledTabellHeader = (width: number, stønadstype: Stønadstype) => {
-    const erOS = stønadstype === Stønadstype.OVERGANGSSTØNAD;
+const utledTabellHeaderBarnetilsyn = (width: number, stønadstype: Stønadstype) => {
+    if (stønadstype !== Stønadstype.BARNETILSYN) {
+        throw Error(`Programeringsfeil - støtter ikke perioder av type: ${stønadstype}`);
+    }
     return width >= 1370 ? (
```

Comment:
Blir veldig mye med denne width-sjekken?
Hadde man kunnet brukt en metode for o finne frem hva som skal renderes?
```js
type BrytTekstType = string | React.ReactElement
const brytTekst = (width: number, tekst: BrytTekstType, kortTekst: BrytTekstType): BrytTekstType => {
    return width >= 1370 ? tekst : kortTekst
}
//...
<Table.HeaderCell scope="col">
   {brytTekst(width, "Periode (fom-tom)", <>Periode <br/> (fom-tom)</>)}
</Table.HeaderCell>
<Table.HeaderCell scope="col">
  <HøyrestiltTekst>
    {brytTekst(width, "Månedsbeløp", <>Måned- <br /> beløp</>)}
  </HøyrestiltTekst>
</Table.HeaderCell>



```

Reply:
Vi bruker &shy; 

:)

Reply:
Måneds&shy;beløp

Reply:
&shy; var fint!